REQUIRE

```diff
@@ -1,3 +1,3 @@
 Compat
 CUDArt
-julia 0.4-
+julia 0.6-
```

Comment:
The fact that I didn't have a chance to test it on 0.5. If it passes there let's relax the restriction.